metadata-json-lint.gemspec

```
@@ -16,13 +16,12 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.5.0'
 
   s.add_runtime_dependency 'json-schema', '>= 2.8', '< 5.0'
-  s.add_runtime_dependency 'semantic_puppet'
+  s.add_runtime_dependency 'semantic_puppet', '~> 1.1'
```

I'd suggest to relax this. AFAIK there's no real reason you need 1.1
```suggestion
  s.add_runtime_dependency 'semantic_puppet', '~> 1.0'
```